### ISSUE TITLE
CURE hyperparameters

### ISSUE THREAD

**Issue by alirezaabdollahpour** [LIONS-EPFL/ELLE#1]
Hi, Thank you a lot for such a nice paper and, moreover, for making your code public.

Can you kindly please explain your hyperparameters for **CURE** experiments?
What lambda (coeff for regularize term) did you use for WRN-28-10 and PRN-18 experiments?

Also, In Figure (1-b) of the paper, you report that with CURE, you can get 43.06% against AA,
Do you mean training the CURE with FGSM adversarial training or just using CURE (because we know that CURE does not have **epsilon**)?

Moreover, these results for Figure (1-b) are for short training (30 epochs) or long training (200 epochs). Which one?

Thanks a lot.

**Comment by megaelius** [comment]
Hi Alireza,

Thanks for your comments! 

- We didn't try CURE in WRN-28-10
- It's true that in the original CURE they don't involve adversarial training, they just regularize curvature. In our experiments we replicate the setup already studied in [GradAlign](https://arxiv.org/abs/2007.02617), i.e., doing single-step AT adding the CURE regularization term.
- For CURE in PRN-18 (Fig 1.b) we use the Short (30 epoch) scheduler with the standard learning rate (0.2), for eps = 8/255 we use the regularization parameter lambda = 1000 and for eps = 16/255 we use lambda = 50000. 

You can use `scripts/run_CURE_cifar10.sh` to replicate our results. Hope that helps!

**Comment by alirezaabdollahpour** [comment]
@megaelius thanks a lot for your quick response. As you mention in the paper, in **short phase training**, you use " cyclic " learning rates. Now, do you mean "cyclic" as "scheduler with the standard learning rate (0.2)?"

Moreover, do you use **CURE** in each epoch? it means from epoch =  0 to epoch = 30?

**Comment by megaelius** [comment]
Thank you for your interest in our work! Please check Fig. 10 in our paper, this is a standard scheduler in the community, we call it the 'GradAlign' scheduler in the code, 0.2 is the maximum learning rate. Yes, we use CURE in every epoch and every training step.

**Comment by alirezaabdollahpour** [comment]
Thanks a lot.
